### DISCUSSION TITLE
Difference between `Bindable<bool>` and `BindableBool`

### DISCUSSION THREAD

**Discussion by nanashi-1** [ppy/osu#21418] [Q&A] [answered]
What is the difference between the two? Is there any advantage to using one over the other? Which one should be used?

**Comment by peppy** [comment, marked as answer]
The latter non-generic versions of bindable classes generally give more control (ie. access to `MinValue` on `BindableNumber` / `BindableInt`). In the case of `BindableBool`, you can access methods like `.Toggle()`.

Check the source code if unsure: https://github.com/ppy/osu-framework/blob/master/osu.Framework/Bindables/BindableBool.cs
